modules/client/examples.md

```diff
@@ -421,13 +421,13 @@ console.log(transfers);
 Handles retrieving DAO asset balances using the DAO address or its ENS domain.
 
 ```ts
-import { AssetBalance, Client, Context } from "@aragon/sdk-client";
+import { TokenBalance, Client, Context } from "@aragon/sdk-client";
```

Comment:
Is there any reason to change this type? I'm fine if the UI is not already using/importing it

Reply:
Just adapting the naming to the subgraph, maybe not the best moment for this change , you are right :)